From d0047ab24e8e92fc2a116b0bccfa10d6b84985be Mon Sep 17 00:00:00 2001
From: Darren Hart <dvhart@linux.intel.com>
Date: Mon, 28 Apr 2014 07:16:46 -0700
Subject: [PATCH 24/24] acpi_lpss: Add Bay Trail pinctrl HID

Add the newer Bay Trail pinctrl HID to the acpi_lpss acpi match table.
Without this, the pinctrl-baytrail.c driver doesn't match the device and
the GPIO are not visible.

PENDING: Out for review
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/acpi/acpi_lpss.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpi_lpss.c b/drivers/acpi/acpi_lpss.c
index 8c2bae980faf..c4666b2c57f7 100644
--- a/drivers/acpi/acpi_lpss.c
+++ b/drivers/acpi/acpi_lpss.c
@@ -173,6 +173,7 @@ static const struct acpi_device_id acpi_lpss_device_ids[] = {
 	{ "80860F14", (unsigned long)&byt_sdio_dev_desc },
 	{ "80860F41", (unsigned long)&byt_i2c_dev_desc },
 	{ "INT33B2", },
+	{ "INT33FC", },
 
 	{ "INT3430", (unsigned long)&lpt_dev_desc },
 	{ "INT3431", (unsigned long)&lpt_dev_desc },
-- 
1.8.1.2

